macOS/README.md

```diff
@@ -0,0 +1,38 @@
+# PlayFab Multiplayer SDK for macOS
+
+## Logging
+
+The PlayFab Multiplayer C++ library includes logging capabilities with a configurable verbosity level. Logging configuration is defined in `PlayFabMultiplayerLogger.json` file which can be deployed as an asset along with the application.
+
+When this file is detected by the application at runtime it will use it to enable logging as configured. The following verbosity levels are currently supported:
+1. `VERBOSE` - everything
+2. `INFO` - less than everything, only important messages and errors
+3. `ERROR` - only errors
+
+The logging is disabled by default, but can be enabled with "enabled" property set to `true`.
+
+### Example configuration file
+
+```json
+{
+    "enabled": false,
+    "bufferSize": 1048576,
+    "maxNumberOfItemsInBatch": 500,
+    "maxBatchWaitTimeInSeconds": 1,
+    "readBufferWaitTimeInMilliseconds": 1,
+    "logFolder": "<any local path>",

```

Comment:
nit: since this is an example, is there any example path that's safe to give (and likely guaranteed to exist on an end user's machine?)